x-pack/legacy/plugins/integrations_manager/public/components/version.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { RequirementVersion } from '../../common/types';
+import euiStyled from '../../../../common/eui_styled_components';
```

Comment:
Please update this for consistency with our other EPM components

```suggestion
import styled from 'styled-components';
```

I see the `infra` plugin uses this pattern, but I don't see the advantage. If you think it's worthwhile, we can open a new ticket and switch patterns in a followup PR.

Can you please replace the other instance of this in `content_collapse` as well?

Reply:
I don't see why you'd create components within the function component if you don't need to.  I don't do it for other components so don't see how a styled one is any different?

Reply:
Are `euiStyled` and `styled` different functions? They look the same to me

Reply:
Sorry, totally misread that comment!